src/main/java/org/openrewrite/quarkus/SLF4JToQuarkusLogger.java

```diff
@@ -0,0 +1,109 @@
+    private static final MethodMatcher LOGGER_TRACE = new MethodMatcher(ORG_SLF_4_J_LOGGER + " trace(String, ..)");
+    private static final MethodMatcher LOGGER_DEBUG = new MethodMatcher(ORG_SLF_4_J_LOGGER + " debug(String, ..)");
+    private static final MethodMatcher LOGGER_INFO = new MethodMatcher(ORG_SLF_4_J_LOGGER + " info(String, ..)");
+    private static final MethodMatcher LOGGER_WARN = new MethodMatcher(ORG_SLF_4_J_LOGGER + " warn(String, ..)");
+    private static final MethodMatcher LOGGER_ERROR = new MethodMatcher(ORG_SLF_4_J_LOGGER + " error(String, ..)");
+
+    @Override
+    public String getDisplayName() {
+        return "Migrate SLF4J Logger injection and usage to Quarkus static `Log`";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Removes usage of SLF4J Logger fields, adjusts imports, and replaces logger method calls with static " +
+                "Quarkus Log calls, including message formatting and method renaming for parameterized logging.";
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
+        return Preconditions.check(
+                new UsesType<>(ORG_SLF_4_J_LOGGER, true),
+                new JavaIsoVisitor<ExecutionContext>() {
+                    @Override
+                    public J.ClassDeclaration visitClassDeclaration(J.ClassDeclaration classDecl, ExecutionContext ctx) {
+                        J.ClassDeclaration cd = super.visitClassDeclaration(classDecl, ctx);
+                        if (cd != classDecl) {
+                            doAfterVisit(new RemoveAnnotationVisitor(new AnnotationMatcher("@lombok.extern.slf4j.Slf4j")));
+                        }
+                        return cd;
+                    }
+
+                    @Override
+                    public J.MethodInvocation visitMethodInvocation(J.MethodInvocation methodInvocation, ExecutionContext ctx) {
+                        J.MethodInvocation mi = super.visitMethodInvocation(methodInvocation, ctx);
+                        if (!LOGGER_TRACE.matches(mi) &&
+                                !LOGGER_DEBUG.matches(mi) &&
+                                !LOGGER_INFO.matches(mi) &&
+                                !LOGGER_WARN.matches(mi) &&
+                                !LOGGER_ERROR.matches(mi)) {
+                            return mi;
+                        }
+
+                        maybeRemoveImport(ORG_SLF_4_J_LOGGER);
+                        maybeAddImport("io.quarkus.logging.Log");
+
+                        List<Expression> args = ListUtils.mapFirst(mi.getArguments(), arg ->
+                                arg instanceof J.Literal && ((J.Literal) arg).getValue() instanceof String ? ((J.Literal) arg)
+                                        .withValue(((String) ((J.Literal) arg).getValue()).replace("{}", "%s"))
+                                        .withValueSource((((J.Literal) arg).getValueSource()).replace("{}", "%s")) : arg);
```

Comment:
We prefer to use `ListUtils` where possible, as that has convenient methods like `mapFirst` to only change the format String, as well as avoiding new collections from being created if there are no changes.